doc/modules/clustering.rst

```diff
@@ -137,8 +137,9 @@ mixture models. KMeans can be seen as a special case of Gaussian mixture
 model with equal covariance per component.
 
 :term:`Transductive <transductive>` clustering methods (in contrast to
-:term:`inductive` clustering methods) are not designed to be applied to new,
-unseen data.
+:term:`inductive` clustering methods, refer to this example
+:ref:`sphx_glr_auto_examples_cluster_plot_inductive_clustering.py`) are not
+designed to be applied to new, unseen data.
```

Comment:
I think the example link should be placed under a subsection. You can scroll down the document a bit to see how it's done for [<code>k_means</code>](https://scikit-learn.org/1.5/modules/clustering.html#k-means), which is rendered as:

<img width="566" alt="k_means_doc_string" src="https://github.com/user-attachments/assets/a0210265-eb87-4f8a-b193-96080577a700">

Reply:
Thank you too @virchan !!! 
I agree, it's better to put the example outside the brackets. I've update the PR, and you can check it now -:)